cmd/skupper/skupper.go

```diff
@@ -109,7 +109,7 @@ func exposeTargetArgs(cmd *cobra.Command, args []string) error {
 		targetType = parts[0]
 	}
 	if !stringSliceContains(validExposeTargets, targetType) {
-		return fmt.Errorf("expose target type must be one of 'deployment', 'statefulset', 'service' or 'pods'")
+		return fmt.Errorf("expose target type must be one of: %s", validExposeTargets)
```

Comment:
Maybe strings.Join would give nicer looking output?

Reply:
yup, sorry for that, now using strings.join and commits squashed